Utilities/sysinfo.cpp

```diff
@@ -142,6 +164,26 @@ std::string utils::get_system_info()
 		}
 	}
 
+	if (has_fma3() || has_fma4())
+	{
+		result += " | FMA";
+		
+		if (has_fma3() && has_fma4())
+		{
+			result += "3+4";
+		}
+
+		else if (has_fma3())
+		{
+			result += "3";
+		}
+
```

Comment:
Here as well.

Also, if you want maybe give both numbers a common prefix, which would simplify the code a bit (e.g. '+3' '+4'). Not really important tho.